activemodel/test/cases/callbacks_test.rb

```diff
@@ -55,33 +55,33 @@ def create
   test "complete callback chain" do
     model = ModelCallbacks.new
     model.create
-    assert_equal model.callbacks, [ :before_create, :before_around_create, :create,
-                                    :after_around_create, :after_create, :final_callback]
+    assert_equal([ :before_create, :before_around_create, :create,
+                                    :after_around_create, :after_create, :final_callback], model.callbacks)
```

Comment:
The indentation here is a bit odd.  How about:

```suggestion
    assert_equal \
      [:before_create, :before_around_create, :create, :after_around_create, :after_create, :final_callback],
      model.callbacks
```